coalib/bears/Bear.py

```diff
@@ -273,6 +277,10 @@ def __init__(self,
                               time limit, use 0.
         :raises TypeError:    Raised when ``message_queue`` is no queue.
         :raises RuntimeError: Raised when bear requirements are not fulfilled.
+        :param debugger       A flag which enable debugger on those bears which
+                              are specified in ``debug_bears`` settings.
+        :param profiler       The Path or name of directory where profiler file
```

Comment:
mismatch with the actual parameter name `profile`. But I guess you should call it `profiler` in the signature, analogous to `debugger`

Reply:
actually, the *cli* argument is `--profile` so I am directly using the `profile` keyword and passing same to bear __init__ , do you want me to change it to `--profiler` or make it `self.profiler = profile`?

Reply:
Ah right. Then keep the name `profile`, however, the docstring has to be corrected ;)